distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSingleSignOn.java

```diff
@@ -119,6 +120,25 @@ public String toString() {
         METADATA, PROTOCOL
     }
 
+    @Before
+    public void setup() throws Exception {
+        // Get all of the metadata
+        String metadata = get(SERVICE_ROOT + "/idp/login/metadata").asString();
+        String ddfSpMetadata = get(SERVICE_ROOT + "/saml/sso/metadata").asString();
+
+        // Make sure all the metadata is valid before we set it
+        validateSaml(metadata, SamlSchema.METADATA);
+        validateSaml(ddfSpMetadata, SamlSchema.METADATA);
+
+
+        // The IdP server can point to multiple Service Providers and as such expects an array.
+        // Thus, even though we are only setting a single item, we must wrap it in an array.
+        setConfig("org.codice.ddf.security.idp.client.IdpMetadata", "metadata", metadata);
```

Comment:
Needed to move these configuration changes to the setup method instead of running once; otherwise, the maps of metadata in IdP server and client could be blank based on the order of tests.